TurkishFilter/sections/specific.txt

```diff
@@ -82,6 +82,7 @@ oyungezer.com.tr##.leftLink
 oyungezer.com.tr##.rightLink
 lezzet.com.tr##.row > div[style="margin-top:-10px"]
 lezzet.com.tr##.banner-box
+lezzet.com##div[id^="medyanet/"]
```

Comment:
`lezzet.com.tr`

Reply:
Also remove these leftovers
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/91165308-19997300-e6d9-11ea-9803-e5abaef225ce.png)

</details><br/>

Reply:
Don't see these.
![image](https://user-images.githubusercontent.com/68852737/91165985-4d28cd00-e6da-11ea-9ccf-89f205f672fd.png)

Reply:
<details><summary>Screenshots:</summary>

![image](https://user-images.githubusercontent.com/33488229/91166330-e0620280-e6da-11ea-9e0d-0a4403ee5063.png)
![image](https://user-images.githubusercontent.com/33488229/91166458-13a49180-e6db-11ea-90ac-d23ba8322251.png)
![image](https://user-images.githubusercontent.com/33488229/91166499-23bc7100-e6db-11ea-8f31-65c8413007df.png)

</details><br/>

Reply:
It seems these are blocked with
![image](https://user-images.githubusercontent.com/68852737/91168156-d2fa4780-e6dd-11ea-81e4-c7c4ab141033.png)
on my side. Should i add these rules to Turkish filters too?

Reply:
Add a specific rule `lezzet.com.tr##ins[data-revive-zoneid]` to the Turkish filter